lib/product-import.coffee

```diff
@@ -136,18 +136,32 @@ class ProductImport
     posts = _.map productsToProcess, (prodToProcess) =>
       existingProduct = @_isExistingEntry(prodToProcess, existingProducts)
       if existingProduct?
-        @_prepareUpdateProduct(prodToProcess, existingProduct).then (preparedProduct) =>
-          synced = @sync.buildActions(preparedProduct, existingProduct)
-          if synced.shouldUpdate()
-            @client.products.byId(synced.getUpdateId()).update(synced.getUpdatePayload())
-          else
-            Promise.resolve statusCode: 304
+        @_fetchSameForAllAttributesOfProductType(prodToProcess.productType)
+        .then (sameForAllAttributes) =>
+          @_prepareUpdateProduct(prodToProcess, existingProduct).then (preparedProduct) =>
```

Comment:
Where do you actually filter out update actions for non master variant attributes?